.github/ISSUE_TEMPLATE/bug.yml

```
@@ -62,22 +65,25 @@ body:
     validations:
       required: true
   - type: dropdown
-    id: sync
+    id: services
     attributes:
-      label: What SDK flavour are you using?
+      label: What Services are you using?
       options:
+        - -- select --
         - Local Database only
         - Atlas Device Sync
-        - Atlas App Services (auth, functions, etc.)
+        - 'Atlas App Services: Auth or Functions etc'
+        - Both Atlas Device Sync and Atlas App Services
     validations:
       required: true
   - type: dropdown
     id: encryption
     attributes:
-      label: Are you using encryption?
+      label: Are you using Encryption?
       options:
-        - Yes, using encryption
-        - No, not using encryption
+        - -- select --
+        - 'Yes'
+        - 'No'
```

```suggestion
        - Yes
        - No
```
Could this also just be a checkbox?

---

No - those can have multiple selections.